prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -64,13 +64,20 @@ def before_run
   end
 
   label def wait
+    leader = postgres_timeline.leader
+    backups = leader.nil? ? [] : postgres_timeline.backups
```

Comment:
```suggestion
    backups = leader ? postgres_timeline.backups : []
```

Generally, if you don't care about the `false`/`nil` difference, it's better not to call `nil?`.